src/components/Drawer/index.tsx

```diff
@@ -0,0 +1,47 @@
+import React, { ReactElement } from 'react';
+import clsx from 'clsx';
+import { Flipped, Flipper } from 'react-flip-toolkit';
+import { useDrawerContext } from './context';
+import { Props } from './types';
+
+
+function Drawer({
+    anchorFrom,
+    children,
+    className,
+    ...otherProps
+}: Props): ReactElement {
+
+
+    const { isOpen, setIsOpen } = useDrawerContext();
+
+    const drawerClasses = clsx(
+        'brew-Drawer',
+        `brew-Drawer--anchorFrom-${anchorFrom}`,
+        { 'brew-Drawer--isOpen': isOpen },
+        className,
+    );
+
+
+    function closeDrawer(): void {
+        setIsOpen(false);
+    }
+
+
+    return (
+        <Flipper flipKey={JSON.stringify([anchorFrom, isOpen, className])}>
+            <Flipped flipId="wrapper">
+                <div className={drawerClasses} {...otherProps}>
```

Comment:
This might be an issue when writing tests, Flipper adds an outer div. To avoid test case confusion, we talked about enclosing Flipper in the drawerClasses